pgxn/neon/libpagestore.c

```diff
@@ -192,8 +192,9 @@ call_PQgetCopyData(char **buffer)
 		{
 			if (!PQconsumeInput(pageserver_conn))
 			{
-				neon_log(LOG, "could not get response from pageserver: %s",
-						 PQerrorMessage(pageserver_conn));
+				char	   *msg = pchomp(PQerrorMessage(pageserver_conn));
```

Comment:
Not entirely sure I have caught on to how whitespace should be handled. I presume this is following a Postgres style guide? I haven't read that yet.

Reply:
Yeah, we use the Postgres style in `pgxn/`. We haven't automated though, so there are many places that violate it in practice.

See https://www.postgresql.org/docs/current/source-format.html for the Postgres style guide. `pgindent`, in `src/tools/pgindent/`, can be used to format code to follow the Postgres style.